Sources/ForageSDK/Foundation/Network/LiveForageService.swift

```diff
@@ -283,12 +243,6 @@ internal class LiveForageService: ForageService {
 // MARK: - Polling
 
 extension LiveForageService: Polling {
-    /// Process Vault Data for polling message.
-    ///
-    /// - Parameters:
-    ///  - vaultResponse: The Vault response.
-    ///  - request: Model element with data to perform request.
-    ///  - completion: Which will return the result.
```

Comment:
removed the duplicated dated/deprecated comments from the implementation file (file was getting bloated). These methods are documented in the `ForageService` protocol.